docs/md/injectors.md

```diff
@@ -0,0 +1,228 @@
+[Main page](../../README.md)
+
+## Enhancing AEM Components' back-end with advanced Injectors
+
+[Injectors](https://sling.apache.org/documentation/bundles/models.html#custom-injectors-1) are a powerful feature of
+Apache Sling that is massively used across modern AEM projects. Injectors pass data directly to the Java classes that
+back AEM components. This reduces boilerplate code and simplifies authoring as well.
+
+Exadel Toolbox Authoring Kit (*ToolKit*) offers a number of Injectors you can use in your projects: both bound to other
+ToolKit features and independent.
+
+#### Injector for EToolbox Lists
+
+You can use it to inject the content of an EToolbox List directly into your Sling model. This injector can be used with
+either a field, a method, or a constructor argument. You can retrieve either a *Collection* / *List* of list items or a
+map depending on the type of the underlying variable (or else the return type of the underlying method).
+
+To get the collection / list of items, use a notation as the following:
+
+```
+@EToolboxList("/content/etoolbox-lists/contentList")
+private List<SimpleListItem> itemsListResource;
+
+```
+
+Note that your injected field's type can be either `List` or an array, like `SimpleListItem[]`. Concerning the type of
+items, you can specify a valid Sling model, or a *Resource*, or else *Object*. In the latter case, *Resource*-s will be
+injected in place of *Object*-s. In the snippets we use the out-of-the-box *SimpleListItem* but any valid Sling model
+will go. Generally, this injector works according to the same principles as the [ListHelper](etoolbox-lists.md) utility
```

Comment:
Perhaps we should specify that Sling Models are supported only if they are adapted from Resource